modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Blogs/Blog.cs

```diff
@@ -0,0 +1,41 @@
+﻿using JetBrains.Annotations;
+using System;
+using Volo.Abp;
+using Volo.Abp.Domain.Entities.Auditing;
+using Volo.Abp.MultiTenancy;
+using Volo.CmsKit.Blogs.Extensions;
+
+namespace Volo.CmsKit.Blogs
+{
+    public class Blog : FullAuditedAggregateRoot<Guid>, IMultiTenant
+    {
+        public Blog(
+            Guid id,
+            [NotNull] string name,
+            [NotNull] string urlSlug,
+            [CanBeNull] Guid? tenantId = null) : base(id)
+        {
+            SetName(name);
+            SetUrlSlug(urlSlug);
+            TenantId = tenantId;
+        }
+
+        public string Name { get; protected set; }
+
+        public string UrlSlug { get; protected set; }
```

Comment:
Instead of naming `UrlSlug`, we can use `Slug` naming everywhere.

Reply:
I leave this to do it later. @ilkayilknur 

Because this change will break multiple branches